lib/services/ios-project-service.ts

```diff
@@ -1312,6 +1326,32 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		}
 		return teamId;
 	}
+
+	private async getEmbeddedPlistData(projectData: IProjectData): Promise<IDictionary<any>> {
+		let embeddedPlistContents = "";
+		const embeddedMobileProvisionPath = path.join(this.getPlatformData(projectData).deviceBuildOutputPath, `${projectData.projectName}.app`, "embedded.mobileprovision");
+
+		embeddedPlistContents = await this.$childProcess.exec(`security cms -D -i ${embeddedMobileProvisionPath}`);
+
+		return plist.parse(embeddedPlistContents);
+	}
+
+	private async getExportOptionsMethod(projectData: IProjectData): Promise<"app-store" | "ad-hoc" | "enterprise" | "development"> {
```

Comment:
Extract this block:
https://github.com/NativeScript/ios-mobileprovision-finder/blob/master/index.ts#L88
onto a "read from file" function and export that function in the ios-mobileprovision-finder
Use it instead of plist parse to obtain the information about the mobileprovision,
If necessary fix/extend the type here: https://github.com/NativeScript/ios-mobileprovision-finder/blob/master/index.ts#L101
then map that type to mode in the current method.